Sources_v3/WebSocketClient/WebSocketClient.swift

```diff
@@ -13,7 +13,7 @@ class WebSocketClient {
     }
     
     /// The notification center `WebSocketClient` uses to send notifications about incoming events.
-    private(set) lazy var notificationCenter: NotificationCenter = environment.createNotificationCenter()
+    let notificationCenter: EventNotificationCenter
```

Comment:
nit: Can we rename this to `eventNotificationCenter` so it's clear it's not just a random notification center?

Reply:
renamed in b043edad1abd90e8ee4e462d7a0bbec9dc176168